packages/blocks/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": "src",
+		"declarationDir": "build-types",
+		"noUnusedParameters": false,
+		"checkJs": false,
+		"noImplicitAny": false
+	},
+	"references": [
+		{ "path": "../autop" },
+		{ "path": "../blob" },
+		{ "path": "../block-serialization-default-parser" },
+		{ "path": "../components" },
+		{ "path": "../compose" },
+		{ "path": "../data" },
+		{ "path": "../deprecated" },
+		{ "path": "../dom" },
+		{ "path": "../element" },
+		{ "path": "../hooks" },
+		{ "path": "../html-entities" },
+		{ "path": "../i18n" },
+		{ "path": "../is-shallow-equal" },
+		{ "path": "../private-apis" }
+	],
+
+	"include": [ "src/**/*", "src/api/i18n-block.json" ]
```

Comment:
what does adding i18n-block do?

Reply:
I think this was because the json wasn't included by default and when `checkJs` was on I was getting an error. (I was typically working on this `checkJs` on so I could see the issues.